src/modules/popup.js

```diff
@@ -18,15 +14,7 @@ const showAndHidePopup = (element, popup) => {
 
 const buildPopup = (event, meals) => {
   const popup = document.querySelector('.popup');
-  console.log(event.target.id);
-  console.log(meals[2]);
-  console.log(meals[event.target.id]);
   const mealitems = Object.entries(meals[event.target.id]);
-  // const ingredientsArray = [];
-  // for (let i = 9; i <= 28; i += 1) {
-
-  // }
```

Comment:
comments that are not used as a title or guide to understand a line of code can be removed to maintain a clean code.  a clean code makes your code readable and easy to understand.